.github/workflows/ci.yml

```diff
@@ -24,7 +24,7 @@ jobs:
         with:
           distribution: 'temurin'
           java-version: ${{ matrix.java }}
-      - uses: gradle/actions/wrapper-validation@v3
-      - uses: gradle/actions/setup-gradle@v3
+      - uses: gradle/actions/wrapper-validation@v4
```

Comment:
```suggestion
```

Reply:
Don't bother with this action?

Reply:
It's now done by default by `setup-gradle`.